packages/sdk/src/addresses/types.ts

```diff
@@ -0,0 +1,8 @@
+export type AddressType = "p2pkh" | "p2sh" | "p2wpkh" | "p2tr";
+
+export type AddressFormat =
+  | "legacy"
+  | "nested-segwit"
+  | "segwit"
+  | "taproot"
+  | "unknown";
```

Comment:
`unknown` was already being used in `getAddressFormat` in the current `ordit-sdk`, adding this type value ensures that we should handle the case in the project where it's `unknown`

Reply:
Makes sense. Just checked, and `unknown` is placed quitely in one of the files. I think it's better if we throw an error if an unknown type is identified.

As I mentioned in the original comment, it would force us to handle this type and we should avoid that. Interestingly no such instances have occurred so far because we are not actively using the Ordit class. 

Now is a good time to remove or we would carry the tech debt baggage in a version that is supposed to be a better version.

Reply:
Removed `unknown` and it now throws an error to the consumer.